frontend/package.json

```diff
@@ -32,8 +34,8 @@
     "typescript": "^4.8.4"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "env-cmd -f ../.env react-scripts start",
```

Comment:
Is this change meant for local dev or deployment?
If we want to include environment variables in Github Actions, we can add our secrets to the repo settings instead.

Reply:
I just made these changes to have just one .env file in root because I was not sure how to make different .env files for frontend and server available through GitHub secrets

Reply:
GitHub Actions doesn't read from .env files anyways, so I think this is needed for that case.